azure-pipelines.yml

```diff
@@ -13,6 +13,18 @@ variables:
   appxPackageDir: '$(build.artifactStagingDirectory)\AppxPackages\\'
 
 steps:
+- task: PowerShell@2
+  inputs:
+    targetType: 'inline'
+    script: |
+      [xml]$manifest = Get-Content $(Build.SourcesDirectory)\src\Notepads\Package.appxmanifest
+      $manifest.Package.Identity.Name="Notepads"
+      $manifest.Package.Properties.DisplayName="Notepads App"
```

Comment:
Do we need this line?

Reply:
Notepads App ?

Reply:
We don't need it anymore, since we aren't changing it. I have removed it.